.gitlab-ci.yml

```diff
@@ -16,12 +16,9 @@ before_script:
     - git fetch origin 'refs/heads/*:refs/heads/*'
     # Get last remaining tags, if any.
     - git fetch --tags origin
+    # Set up git identity
+    - git config --global user.name "Northern.tech" && git config --global user.email "info@northern.tech"
 
-    - git clone git://github.com/mendersoftware/mendertesting
-
-script:
-    # Check commit compliance.
-    - mendertesting/check_commits.sh
```

Comment:
This check we shouldn't remove.

The modern way of having it is just importing it from `mendertesting`. See [here](https://github.com/mendersoftware/mender-convert/blob/next/.gitlab-ci.yml#L7-L8) for an example.